hyperspy/tests/io/test_hdf5.py

```diff
@@ -289,6 +289,25 @@ def tearDown(self):
         remove(self.filename)
 
 
+class TestAxesConfiguration:
+
+    def setUp(self):
+        self.filename = 'testfile.hdf5'
+        s = BaseSignal(np.zeros((2, 2, 2, 2, 2)))
+        s.axes_manager.signal_axes[0].navigate = True
+        s.axes_manager.signal_axes[0].navigate = True
```

Comment:
No, I was just wondering, because I didnt' see it in the documentation at first. As I said I personally like this approach much more than the previous one (why should the last axis be special by default?).
Overall (again, just my opinion), it's more intuitive now :-)